chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java

```diff
@@ -640,13 +646,24 @@ public void getMessages(@NonNull final Conversation conversation,
             limitCount = GET_MESSAGES_DEFAULT_LIMIT;
         }
 
+        cacheController.getMessages(conversation, limitCount, before, order, new GetCallback<List<Message>>() {
+            @Override
+            public void onSucc(@Nullable List<Message> object) {
+              // TODO: pass the cached result to callback
```

Comment:
intentionally left out?

Reply:
Yes.

I haven't decided the pattern of returning cached result yet. Unlike iOS, Android SDK does not have cached query, so nothing for reference. I want to leave this part later.